ledfx/core.py

```diff
@@ -314,43 +319,49 @@ async def async_stop(self, exit_code):
             return
 
         print("Stopping LedFx.")
+        try:
+            # 1 = Error
```

Comment:
should we get rid of this comment block now in favour of the code implementation above with strings? 1 point on maintanance etc.